source/new-ia/by-role/system-admin/reverse-proxy-configuration.rst

```diff
@@ -0,0 +1,75 @@
+:is-up-to-date: True
+:since-version: 4.0.3
+:nosearch:
+
+.. index:: Reverse Proxy Configuration, Configuring Apache Virtual Host, Configuring Apache vhost
+
+.. _newIa-reverse-proxy-configuration:
+
+===========================
+Reverse Proxy Configuration
+===========================
+
+.. version_tag::
+   :label: Since
+   :version: 4.0.3
+
+In this section, we discuss how to configure a reverse proxy for Apache for your
+CrafterCMS authoring and delivery installs.
+
+Below are the directives used for setting up a reverse proxy with Apache:
+
+.. code-block:: apache
+   :caption: *Authoring Configuration*
+
+   <VirtualHost *:80>
+     ProxyPreserveHost On
+
+     # Authoring and Preview
+     ProxyPassMatch ^/(studio/events)$  ws://localhost:8080/$1
+     ProxyPass / http://localhost:8080/
+     ProxyPassReverse / http://localhost:8080/
+
+     ErrorLog ${APACHE_LOG_DIR}/crafter-studio-error.log
+     CustomLog ${APACHE_LOG_DIR}/crafter-studio-access.log combined
+   </VirtualHost>
+
+
+.. code-block:: apache
```

Comment:
This is very different from the one in the prior file. We can meet with Russ and Alfonso and decide which one to use.

Also, why is this config in two place in new IA?